tasks/pkg-redhat.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Download new RPM key
+  get_url:
+    url: "http://yum.datadoghq.com/DATADOG_RPM_KEY.public.E09422B3"
+    dest: /tmp/DATADOG_RPM_KEY.e09422b3.public
```

Comment:
Does this mean this file will always be there, even when the `rpm` key is already imported ?